docs/_docs/04_reference/02-cli-options.md

```diff
@@ -341,6 +341,12 @@ Will create the following directory structure:
     └── terragrunt.hcl
 ```
 
+**Note**: Stack generation is done in parallel. You can control the number of parallel tasks using the `--parallelism` CLI flag:
```

Comment:
(by default, generation will run with x parallel generators at a given time)

Reply:
DefaultParallelism = math.MaxInt32

Reply:
Added note that by default it is limited by GOMAXPROCS